package.json

```
@@ -41,23 +32,29 @@
   "devDependencies": {
     "@babel/cli": "^7.23.9",
     "@babel/core": "^7.23.9",
+    "@babel/plugin-transform-typescript": "^7.25.9",
     "@commitlint/cli": "^18.6.0",
     "@commitlint/config-conventional": "^18.6.0",
     "@reduxjs/toolkit": "^2.1.0",
-    "amex-jest-preset": "^7.0.0",
+    "@types/redux-immutable": "^4.0.6",
+    "@typescript-eslint/eslint-plugin": "^6.21.0",
+    "@vitest/coverage-v8": "^2.1.4",
     "babel-preset-amex": "^4.0.3",
     "coveralls": "^3.1.1",
     "eslint": "^8.56.0",
     "eslint-config-amex": "^16.0.0",
+    "eslint-import-resolver-typescript": "^3.6.3",
     "eslint-plugin-jest": "^27.6.3",
     "eslint-plugin-jest-dom": "^4.0.3",
     "husky": "^8.0.3",
     "immutable": "^4.3.5",
-    "jest": "^29.7.0",
     "lockfile-lint": "^4.12.1",
+    "prettier": "^2.8.8",
     "redux": "^5.0.1",
     "redux-immutable": "^4.0.0",
-    "rimraf": "^5.0.5"
+    "rimraf": "^5.0.5",
+    "typescript": "^5.4.4",
```

Typescript doesn't follow semver because every change is breaking.
```suggestion
    "typescript": ">=5.4.4",
```

---

Perhaps `~`? Or are even patch versions breaking?

---

Yeah since it's a dev dep it honestly doesn't really matter.

---

addressed in [f04d16d](https://github.com/americanexpress/vitruvius/commit/f04d16dcc861948b2ea96c64c987dda8dd8c7c75)